Tools/ardupilotwaf/boards.py

```diff
@@ -223,6 +223,10 @@ def configure_env(self, cfg, env):
         ]
         env.LIB += cfg.env.LIB_RT
 
+        cfg.check_cfg(package='libsystemd', mandatory=False, global_define=True,
+                      args = ['--libs', '--cflags'])
```

Comment:
I actually think a function here adds unnecessary noise. Here we have a single call that's not supposed to be used elsewhere. check_librt does a little bit more.